101-virtual-wan/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {

```

Comment:
remove these and just use defaultValues in the template (most are already there)